addons/events/index.js

```diff
@@ -1,60 +1,8 @@
-const AddonsApi = require('modulor-storybook/addons');
+const AddonsApi = require('../../addons');
 const dateFormat = require('dateformat');
+const stylesTemplate = require('./style.css').default;
+require('inspector-component/src/index.js');
```

Comment:
is it possible to define `index` section in `inspector-component` package.json? looks a bit dirty this way

Reply:
I am still improving the inspector; for now pointing it to src is best bet because it is also deliverable through bower, it points to `dist` as main file.

Reply:
ok then